src/main/java/org/tron/common/runtime/Runtime.java

```diff
@@ -512,6 +513,7 @@ public void go() {
           result.getDeleteAccounts().clear();
           result.getLogInfoList().clear();
           result.resetFutureRefund();
+          result.rejectInternalTransactions();
 
           if (result.getException() != null) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this code to not nest more than 3 if/for/while/switch/try statements. ([squid:S134](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS134))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1633&issues=AWak_C2vdPPRVdwyhXLG&open=AWak_C2vdPPRVdwyhXLG)

[ISSUE_KEY]: AWak_C2vdPPRVdwyhXLG